app/templates/about.jade

```diff
@@ -1,188 +1,380 @@
+              a.job-link.btn.btn-lg.btn-navy(href="https://jobs.lever.co/codecombat/2d0194ed-7c69-4aab-b1f9-363ca4b9f04b")
+                | Learn More
+          .col-sm-6.col-md-5.col-lg-4
+            .job-listing
+              h5 Sales Representative
+              .text-center
+                small.label
+                  | San Francisco • Fulltime
+              p.small School districts are scrambling to offer computer science classes to all their students as a core subject. They have had no solution, because they can't afford to hire enough programming teachers – until now. 
+              a.job-link.btn.btn-lg.btn-navy(href="https://jobs.lever.co/codecombat/3f6ff123-16ce-4ecb-aba3-dcf4e8927c47")
+                | Learn More
+          .col-sm-6.col-md-5.col-md-offset-1.col-lg-4.col-lg-offset-0
+            .job-listing
+              h5 Create Your Own
+              .text-center
+                small.label
+                  | San Francisco • Fulltime
+              p.small Are you passionate about CodeCombat but don't see a job listed that matches your qualifications? Write us and show how you think you can contribute to our team. We'd love to hear from you!
+              p.small
+                | Send us a note at 
+                a(href="mailto:team@codecombat.com") team@codecombat.com 
+                | introducing yourself and we might get in touch in the future!
+        
+      #contact.anchor
+        h3 Press & Contact
+        h4 Need more information? Get in touch with us at team@codecombat.com
+        
+      #files
+        .row
+          .col-md-8
+            #screenshots
+              span.label Game Screenshots
+              //TODO: This makes spacing weird. 
+              //      How should I do it nicely with correct spacing, wrapping,
+              //      and no flexboxes?
+              .hidden-sm.hidden-md.hidden-lg
+                small
+                  | (click to view full size)
+              #screenshot-grid
+                a(data-toggle="modal", data-target="#screenshot-lightbox")
+                  img(src="/images/pages/about/Screenshot_Desert.png")
+                a(data-toggle="modal", data-target="#screenshot-lightbox")
+                  img(src="/images/pages/about/Screenshot_Forest.png")
+                a(data-toggle="modal", data-target="#screenshot-lightbox")
+                  img(src="/images/pages/about/Screenshot_Dungeon.png")
+                a(data-toggle="modal", data-target="#screenshot-lightbox")
+                  img(src="/images/pages/about/Screenshot_Glacier.png")
+              .clearfix.hidden-xs
+                small
+                  | (click to view full size)
+          .col-md-4
+            #downloads-container
+              #downloads
+                .label Download Assets & Information
```

Comment:
These two columns should be 6 and 6 at small width, otherwise the right column looks smooshed.